main/newscorm/learnpath.class.php

```diff
@@ -10691,6 +10723,109 @@ public function createForum($forumCategoryId)
         return $forumId;
     }
 
+    private function getFinalItem()
```

Comment:
This requires a PHPDoc block. The name of the function is too generic, but even if it wasn't, every function needs to be documented.